CHANGES.rst

```diff
@@ -6,7 +6,7 @@
 ==================
 
 - Add zope.lifecycleevent subscribers. You must include ``subscribers.zcml``
-  to use these. See https://github.com/zopefoundation/zc.intid/issues/5
+  to use these and have :mod:`zope.intid` installed. See :issue:`5`.
```

Comment:
I don't know from the `:issue:` Sphinx role:  is it from an extension?  I don't see one [enabled](https://github.com/zopefoundation/zc.intid/blob/master/doc/conf.py#L32-L38) which might add it.